qt/scientific_interfaces/Muon/ALCBaselineModellingView.h

```diff
@@ -30,11 +32,11 @@ class MANTIDQT_MUONINTERFACE_DLL ALCBaselineModellingView : public IALCBaselineM
   ALCBaselineModellingView(QWidget *widget);
   ~ALCBaselineModellingView() override;
 
+  void subscribePresenter(ALCBaselineModellingPresenter *presenter) override { m_presenter = presenter; }
```

Comment:
Ideally we want to subscribe an `IALCBaselineModellingPresenter` (an interface class presenter) to the view. This would give extra flexibility in the future if we wanted to create another Presenter with a different implementation to the current Presenter. As long as the new Presenter adhered to the same interface, we could reuse the same view but with different business/gui logic (in the presenter)

Reply:
Would you like that changed for this pr? or is this just a comment.

Reply:
If you are happy to do it in this PR then yes please!

Reply:
Should all the method from the current presenter be in the interface? or just some of them?

Reply:
Only the methods which are called from the view or from the ALCInterface file. The rest can probably be private I think